src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -40,30 +40,31 @@
 #include "Log.h"
 #include "LootMgr.h"
 #include "MotionMaster.h"
+#include "MovementPacketSender.h"
+#include "MovementPackets.h"
 #include "MoveSpline.h"
 #include "MoveSplineInit.h"
+#include "MoveSplineInit.h"
```

Comment:
Heh. All I did here was SORT the includes, it means it was already duplicated before